internal/appsec/appsec.go

```diff
@@ -0,0 +1,197 @@
+package appsec
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"sync"
+	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/internal/intake"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/internal/intake/api"
+	httpprotection "gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/internal/protection/http"
+	appsectypes "gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/types"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/dyngo"
+)
+
+type (
+	Config struct {
```

Comment:
Please consider that we ultimately want to document everything thoroughly and clearly before publishing this. It is important for any new comer to be able to ramp up easily simply by reading the documentation.

Reply:
Fixed: the pubic API is now completely commented.